how-to/get-started/quick-start-with-tidb.md

```diff
@@ -0,0 +1,320 @@
+---
+title: Quick Start with TiDB
+summary: Learn how to quick start your first TiDB cluster and try out the features.
+category: how-to
+---
+
+# Quick Start with TiDB
+
+This document describes how to try out TiDB, a distributed SQL database. You can choose any of the following three methods to start your first TiDB cluster:
+
+- [Try out TiDB with one click using TiDB-Wasm](#try-out-tidb-with-one-click-using-tidb-wasm)
+- [Deploy a local test environment using TiUP Playground](#deploy-a-local-test-environment-using-tiup-playground)
+- [Deploy a quasi-production test environment on a single machine using TiUP cluster](#deploya-quasi-production-test-environment-on-a-single-machine-using-tiup-cluster)
+
+> **Note:**
+>
+> It is not recommended to deploy TiDB for production environments with the above three methods. Instead, you can [deploy TiDB using TiUP](/how-to/deploy/orchestrated/tiup.md).
+
+## Try out TiDB with one click using TiDB-Wasm
+
+- Scenario: Have a first taste on the basic features of TiDB, such as syntax and compatibility.
+- Time required: Instant
+
+TiDB-Wasm is a TiDB cluster running in the browser. You can use TiDB simply by opening a web page and perform basic operations such as executing SQL statements and verifying compatibility.
```

Comment:
```suggestion
TiDB-Wasm is a TiDB cluster running in the browser. With TiDB-Wasm, you can use TiDB simply by opening a web page and perform basic operations such as executing SQL statements and verifying compatibility.
```

Reply:
-> TiDB-Wasm enables you to run a TiDB server instance in a browser so that you can experiment with SQL and test TiDB compatibility with MySQL queries.